packages/clerk-js/src/ui/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -209,15 +211,17 @@ const CountryCodeListItem = memo((props: CountryCodeListItemProps) => {
   );
 });
 
-export const PhoneInput = forwardRef<HTMLInputElement, PhoneInputProps>((props, ref) => {
-  // @ts-expect-error
-  const { __internal_country } = useClerk();
+export const PhoneInput = forwardRef<HTMLInputElement, PhoneInputProps & { feedbackType?: FeedbackType }>(
```

Comment:
Can we move the `& { feedbackType?: FeedbackType }` on the type definition at the top of the file?